src/Microsoft.Tye.Hosting/ProcessRunner.cs

```diff
@@ -168,6 +168,14 @@ private void LaunchService(Application application, Service service)
             // Set by BuildAndRunService
             var args = service.Status.Args!;
             var path = service.Status.ExecutablePath!;
+
+            if (_options.Watch && service.ServiceType == ServiceType.Project)
+            {
+                path = "dotnet";
+                // Don't use launch settings as we already configured ports earlier.
+                args = $"watch run {service.Status.ProjectFilePath} --no-launch-profile " + args;
```

Comment:
This isn't great because watch launches another process no? We want direct tracking of the process being launched. Test what happens when you kill the process.

Reply:
Yeah you are right, this is tracking the watch process.

Reply:
Looks like I couldn't get away with the easy path 😄

Reply:
That's why I didn't do it yet 😄